src/variables.jl

```diff
@@ -1104,15 +1106,17 @@ function relax_integrality(model::Model)
                 set_integer(v)
             elseif info.binary
                 set_binary(v)
-                if info.has_lb
-                    set_lower_bound(v, info.lower_bound)
-                else
-                    delete_lower_bound(v)
-                end
-                if info.has_ub
-                    set_upper_bound(v, info.upper_bound)
-                else
-                    delete_upper_bound(v)
+                if !info.has_fix
```

Comment:
What happens if a binary variable is fixed to a value outside of [0, 1]? This should trigger an error, IMO.

Reply:
Using `relax_integrality` should trigger an error if a binary variable as been wrongly fixed? Shouldn't that be checked by `fix`?

Reply:
> Using relax_integrality should trigger an error if a binary variable as been wrongly fixed?

Using `relax_integrality` should trigger an error if the relaxation of a variable's domain is an empty set.

> Shouldn't that be checked by fix?

Not really, JuMP allows you to write models that are inconsistent (e.g., infeasible). This isn't checked when the constraints are added but later on.

Reply:
To be honest, I am not sure that throwing an error in `relax_integrality` is something that the user will expect. I tried something different to deal with fixed binary variables. It's obviously not perfect, yet it will behave as I would expect: the relaxation of this kind of infeasible model remains infeasible.

If you still think an error is the way to go, then I'll add that but I might need a little help to get a comprehensible sentence.